### DISCUSSION TITLE
fasUnlink

### DISCUSSION THREAD

**Discussion by ecotrissy** [quasarframework/quasar#12692] [Quasar App Extensions] [answered]
`import { fasUnlink } from '@quasar/extras/fontawesome-v5' // "fas fa-unlink"`

This icon doesn't seem to work (other similar ones are working).

**Comment by MilosPaunovic** [comment, marked as answer]
It works here [https://iconexplorer.app/icons/fontawesome-v5?filter=unlink](https://iconexplorer.app/icons/fontawesome-v5?filter=unlink)

Could you provide code reproduction?

**Reply by ecotrissy** [reply]
Thank you!
My bad, it was a typo.
